src/model/config.rs

```diff
@@ -110,6 +110,7 @@ impl StorageConfig {
                     region: get_var("REGISTRY_S3_REGION")?,
                     access_key: get_var("REGISTRY_S3_ACCESS_KEY")?,
                     secret_key: get_var("REGISTRY_S3_SECRET_KEY")?,
+                    root: get_var("REGISTRY_S3_ROOT").unwrap_or_default(),
```

Comment:
The `unwrap_or_default` is here to make setting this env variable optional, but could also require it and as for the keys and empty string would be 'do the default thing of going to the top-level'.